src/core/a-scene.js

```diff
@@ -721,27 +723,31 @@ var AScene = module.exports = registerElement('a-scene', {
      * Renders with request animation frame.
      */
     render: {
-      value: function (t) {
+      value: function (time) {
         var camera = this.camera;
         var stats = this.stats;
+        var timeDelta = time - this.time;
 
         if (stats) {
           stats('rAF').tick();
           stats('FPS').frame();
         }
-        TWEEN.update(t);
+
+        TWEEN.update(time);
```

Comment:
Tweens should not run if the scene is paused. Can we put it inside the if statement below?